homeassistant/components/meteo_france/const.py

```diff
@@ -51,7 +51,7 @@
         ENTITY_UNIT: UNIT_PERCENTAGE,
         ENTITY_ICON: "mdi:weather-snowy",
         ENTITY_CLASS: None,
-        ENTITY_ENABLE: False,
+        ENTITY_ENABLE: True,
         ENTITY_API_DATA_PATH: "probability_forecast:snow:3h",
     },
     "freeze_chance": {
```

Comment:
Maybe we should enable by default `snow` and `freeze` only on mountain cities/departements.
What do you think ?

Reply:
What is the rational to not enable it ? There is no other usage of this value.

Reply:
To not create sensors that the user won't need, and not record data from it in the database.

Reply:
I think if we put it not enable by default a lot of user won't understand it could be activated.

Reply:
I prefere to keep like this.